deploy/helm/sumologic/templates/setup/setup-configmap.yaml

```diff
@@ -128,6 +130,11 @@ data:
         collector_id = "${sumologic_collector.collector.id}"
     }
 
+    resource "sumologic_http_source" "traces_source" {
```

Comment:
I think we are exposing new source from the collection side so this is not right maybe? becuase it will be different from sumologic_http_source